.github/workflows/dummy.yml

```diff
@@ -1,6 +1,6 @@
 name: Dummy-echo
 
-on: [push]
+on: [push,pull_request]
```

Comment:
Looks like Github doesn't care if a modification of a workflow happens in a fork and tells to run events on a PR. This is really dangerous. We need to wait for some extra protection of workflows before using them with self hosted runners.

https://github.com/ginkgo-project/ginkgo/runs/1051608479